modin/core/storage_formats/base/query_compiler.py

```diff
@@ -32,6 +32,7 @@
 )
 from modin.error_message import ErrorMessage
 import modin.core.storage_formats.base.doc_utils as doc_utils
+from modin.modin_logging import LoggerMetaClass
```

Comment:
```suggestion
from modin.logging import LoggerMetaClass
```
Do you think this would be better?

Reply:
This was the initial name, but when running jupyter notebooks the name conflicts with JupyterLab and breaks which was the reason for renaming it.

Reply:
@jeffreykennethli Could you describe the error you ended up getting here?

Reply:
Here was the original comment: https://github.com/modin-project/modin/pull/4372#issuecomment-1117825144

I can confirm the above issue with JupyterLab still exists (I tested by renaming the module locally on the latest commit). Here is the debug output, where it appears JupterLab tries to import modin logging and running `getLogger` on a file import instead of the python logging module.

https://gist.github.com/jeffreykennethli/371e70b2d938143ec591e8caf3c4ae84

Reply:
@jeffreykennethli is this because you were working in the same directory as the `modin.logging` directory? Does this same issue occur elsewhere?

Calling this `modin_logging` is really clunky and I prefer `logging`

Reply:
I hadn't thought of that, but yes, I can confirm this only happens when working from the repo's base dir and `modin`, but not any other dir, not even `modin.logging`.

I do agree that `modin_logging` is clunky. Perhaps we can use `logging` and have this edge case, or `logger`?

Reply:
@devin-petersohn Thoughts on this? I think `logger` is fine.